react/week1/react-w1/src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import ToDo from "./App";
+import toDOLists from "./toDofile";
+const todos = toDOLists();
+
+function ToDoTable(props) {
+  return <div>
+    <h1>ToDo List:</h1>
+    <ul>
+      {todos.map((todo) => {
+        return <ToDo description={todo.description} endDate={todo.endDate} />;
+      })}
+    </ul>
```

Comment:
Just a small comment:
The console is showing a warning that says `Each child in a list should have a unique "key" prop.`

you can avoid the problem by simply providing a uniqe `key` that have the value of `todo.id` (you of course need to add `id` to each todo in `toDofile` ):

```suggestion
    <ul>
      {todos.map((todo) => {
        return <ToDo  key={todo.id} description={todo.description} endDate={todo.endDate} />;
      })}
    </ul>
```

You will learn more about it in the next weeks :)

Reply:
Yes i was wondering why was this error. Thanks for this